sqlmesh/schedulers/airflow/operators/mysql.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+import typing as t
+
+from airflow.models import BaseOperator
+from airflow.providers.mysql.hooks.mysql import MySqlHook
+from airflow.utils.context import Context
+
+from sqlmesh.schedulers.airflow.operators.targets import BaseTarget
+
+
+class SQLMeshMysqlOperator(BaseOperator):
```

Comment:
Nit - let's match existing naming convention for MySQL:
```suggestion
class SQLMeshMySQLOperator(BaseOperator):
```

Reply:
just updated the code, I want to know how could I test this operator with airflow in local ?